### ISSUE TITLE
Add clean up disk space feature

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-online-tools#328]
The application is taking an increasing amount of disk space on the production instance.

Most of the space is consumed in the /media folder - /media/AnonymousUser being the largest at 2GB.

Suggest we add some type of regular clean-up of the media folders.  This could be added into the app itself or a separate CRON job we could install via the Docker files.

**Comment by rtgdk** [comment]
The folder stores the license files uploaded by the user (for validating/comparing/etc). Old files are generally not needed. When user are not logged in, the files are uploaded to `AnonymousUser`. I think cron job should delete more than 10 days old files. The api also creates some files which are returned to the user for downloading.

**Comment by rtgdk** [comment]
@autoantwort No files are not needed after the processing is done. We are saving the uploaded files to be used by the spdk java tools application. If there's a way we can instead send the file as a stream to the java application then we can use that. Or we can add cleanup check after the processing is done. Or a separate cleanup cron job

**Comment by BassCoder2808** [comment]
Hi @goneall I would be happy to contribute to this issue if it is still open.
Going by the approach which @rtgdk mentioned above, I have made a small CRON job for the same, let me know if it is correct or needs any improvement if this issue is till open
```
import os
import time

MEDIA_DIR = '/media/AnonymousUser'
DAYS_THRESHOLD = 10

now = time.time()

for filename in os.listdir(MEDIA_DIR):
    filepath = os.path.join(MEDIA_DIR, filename)
    if os.path.isfile(filepath) and (now - os.stat(filepath).st_mtime) > (DAYS_THRESHOLD * 86400):
        os.remove(filepath)

```

**Comment by goneall** [comment]
@BassCoder2808 - Thanks!  The above approach looks good to me.

We could add the script in the `scripts` folder and update the [dockerfile.prod](https://github.com/spdx/spdx-online-tools/blob/main/dockerfile.prod) file to install and run the script.

Not quite sure the best way to test the changes before pushing to production - let me know if you have any ideas.

Please feel free to create a PR.

**Comment by BassCoder2808** [comment]
Sure @goneall I will create a pr for the same, and will also try how we can check if the code works correctly or not

**Comment by BassCoder2808** [comment]
@goneall I have written test case for the same, let me know your thoughts on the same, I have not added this in the above pr
```
import datetime
import os
from django.test import TestCase

class TestCronJob(TestCase):
    def test_delete_old_files(self):
        # create a test directory with some files
        test_dir = os.path.join(os.getcwd(), 'test_dir')
        os.makedirs(test_dir, exist_ok=True)
        for i in range(1, 11):
            file_path = os.path.join(test_dir, f'test_file_{i}.txt')
            with open(file_path, 'w') as f:
                f.write('test')
            # set file creation time to 11 days ago
            creation_time = datetime.datetime.now() - datetime.timedelta(days=11)
            os.utime(file_path, (creation_time.timestamp(), creation_time.timestamp()))

        # run the cron job to delete old files
        # (you may need to modify this depending on how you're running the cron job)
        os.system('python manage.py delete_old_files')

        # check that only files older than 10 days were deleted
        for i in range(1, 11):
            file_path = os.path.join(test_dir, f'test_file_{i}.txt')
            if i <= 5:
                self.assertFalse(os.path.exists(file_path), f'{file_path} should have been deleted')
            else:
                self.assertTrue(os.path.exists(file_path), f'{file_path} should not have been deleted')
````

**Comment by goneall** [comment]
@BassCoder2808 - the above looks like a good approach to testing the feature.

**Comment by rtgdk** [comment]
@BassCoder2808 Let's add the test as well to the PR

**Comment by BassCoder2808** [comment]
Sure @rtgdk I will add the test as well to the pr

**Comment by BassCoder2808** [comment]
Hi @rtgdk I have added the unit test as well to the PR now and it is passing as well 
![image](https://user-images.githubusercontent.com/65075935/224781268-0a5013fc-e82a-4d69-a9f4-450332f34ae2.png)

**Comment by goneall** [comment]
Resolved with PR #440
